src/deps_compat.jl

```diff
@@ -23,18 +24,29 @@ to the corresponding `check_\$x` keyword argument.
   `@test` for "deps".
 - `ignore::Vector{Symbol}`: names of dependent packages to be ignored.
 """
-function test_deps_compat(pkg::PkgId; check_extras = true, check_weakdeps = true, kwargs...)
+function test_deps_compat(
+    pkg::PkgId;
+    check_julia = true,
+    check_extras = true,
+    check_weakdeps = true,
+    kwargs...,
+)
+    if check_julia !== false
```

Comment:
```suggestion
    check_julia::Bool = true,
    check_extras::Bool = true,
    check_weakdeps::Bool = true,
    kwargs...,
)
    if check_julia
```

I think argument-type specification is better for readability and error handling.

Reply:
That is not possible here in an easy way, as one can pass a `Bool` or a `NamedTuple`. The docstring says `A NamedTuple can be used to pass keyword arguments with test options (see below).`